bolt/src/main/java/com/slack/api/bolt/AppConfig.java

```diff
@@ -29,14 +29,34 @@ private EnvVariableName() {
         public static final String SLACK_BOT_TOKEN = "SLACK_BOT_TOKEN";
         public static final String SLACK_SIGNING_SECRET = "SLACK_SIGNING_SECRET";
         public static final String SLACK_VERIFICATION_TOKEN = "SLACK_VERIFICATION_TOKEN";
+
+        public static final String SLACK_CLIENT_ID = "SLACK_CLIENT_ID";
```

Comment:
Change the default env variable names to align with Bolt JS/Python. The old ones are still live.